doc/toolbox-run.1.md

```diff
@@ -0,0 +1,59 @@
+% toolbox-run(1)
+
+## NAME
+toolbox\-run - Run a command in an existing toolbox container
+
+## SYNOPSIS
+**toolbox run** [*-c | --container NAME*] [*-r | --release RELEASE*] [*COMMAND*]
```

Comment:
Would be nice to put the longer variant first for consistency with the Podman manuals. The `--help` output is the other way round, but that should be replaced with the manuals like `git help`.